docs/source/getting_started/installation.md

```diff
@@ -152,13 +192,13 @@ If you have Docker installed, you can use JupyterLab and Elyra by running one of
 Images can be pulled from [Docker Hub](https://hub.docker.com/r/elyra/elyra/tags) 
 
 ```
-docker pull elyra/elyra:1.4.1
+docker pull elyra/elyra:2.0.1
 ```
```

Comment:
again, I believe the master branch should be `dev` otherwise the release script is not updating this...

Reply:
Thank you for the clarification during the scrum. To recap, `elyra:dev` is a placeholder that is the valid tag for the master branch. When a release is published the placeholder is replaced with the release's tag, like so:  `elyra:[version-tag]`